cjaasPythonClient/generate_saas_token.py

```diff
@@ -35,12 +35,14 @@ def generate_token(cls, param):
             raise ValueError("Both validityDays and validityHours can't be 0")
         valid_until = (datetime.utcnow() + timedelta(days=param.validity_days) + timedelta(
             hours=param.validity_hours)).isoformat(timespec='milliseconds')
-        token = "st={" + param.tenant + "}&so={" + param.org + "}&ss={" + param.service + \
-                "}&sp={" + param.permission + "}&se={" + valid_until + "Z}&sk={" + param.key_name + "} "
+        token = "st=" + param.tenant + "&so=" + param.org + "&ss=" + param.service + \
+                "&sp=" + param.permission + "&se=" + \
+            valid_until + "Z&sk=" + param.key_name
         message = bytes(token, 'utf-8')
         secret = bytes(param.tenant_key, 'utf-8')
-        signature = base64.b64encode(hmac.new(secret, message, digestmod=hashlib.sha256).digest())
-        # print(signature)
+        signature = base64.b64encode(
+            hmac.new(secret, message, digestmod=hashlib.sha256).digest())
+        print(token + "&sig=" + signature.decode("utf-8"))
```

Comment:
i have been thinking of late, if we should also include the SharedAccessSignature word as part of the output token. so it is easy for copy paste into rest api calls. should we also assign this print output as return value. pretty much a token.

Reply:
made the change myself. merging this in